samples/ots16/event.json

```diff
@@ -20,38 +20,6 @@
     "organizer_name": "FOSSASIA",
     "privacy": "public",
     "schedule_published_on": "2016-05-04 09:00:00",
-    "social_links": [
-        {
```

Comment:
Why did you remove social_links ? Import JSON files should reflect the APIs and event GET API has social_links so I think we should keep it. 
Also the export of event will have social_links field in event.json (as they are based on APIs ) .